mobify/astro-scaffold
# [HYB 689] Scaffold Navitron

Adding Navitron into the Astro Scaffold!

JIRA: [**[HYB-689] As a scaffold user, I want a flyout left drawer**](https://mobify.atlassian.net/browse/HYB-689)
Linked PRs: **None**

## Changes
- Navitron added to scaffold
- Changed folder structure for `scaffold-www`
- Error modal changed to support both layout architectures
- `menuItems.js` has different object structure to support navitron
- `tabItems.js` added for configuring tab items

## How to test-drive this PR
**NOTE: May require npm linking the latest develop branch of Astro**
- Run app with drawer layout on iOS, make sure navitron replaces left menu
- Test out navitron navigation on iOS/Android
- with Console hooked up, call RPC on the worker to re-render menu. Ex:
```
Astro.jsRpcMethod('renderLeftMenu', ['menuItems'])([
    {
        title: '1',
        subItems: [
            {
                title: '2',
                href: '/accessories'
            }
        ]
    }
]);
```
- Run tab layout on iOS, make sure that it works as it did before.

## TODOS:
